pkgs/unified_analytics/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 6.1.4-wip
 
 - Fix formatting and remove dependency on `package:intl`.
+- Add more parameters (androidAppId and iosBundleId) in devtools events.
```

Comment:
super nit: use back tics and code references

 - Added `androidAppId` and `iosBundleId` to the `Event.devtoolsEvent` constructor.